exporter/opencensusexporter/metadata.yaml

```diff
@@ -5,3 +5,5 @@ status:
   stability:
     beta: [traces, metrics]
   distributions: [contrib, observiq]
+  codeowners:
+    active: [open-telemetry/collector-approvers]
```

Comment:
Can you double-check this one? If this should be a group, shouldn't the group be added to all components?

Reply:
Yes, see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS#L71

The core approvers are apparently codeowners of this component.

Please note the group `@open-telemetry/collector-contrib-approvers` will be added to all components in any case.

Reply:
Ohhh, I missed that this was core's approvers, not contrib. Sounds good, then.